src/MarginTrading.Brokers/MarginTrading.AccountMarginEventsBroker/Application.cs

```diff
@@ -38,19 +39,19 @@ protected override Task HandleMessage(AccountMarginEventMessage message)
                 AccountId = message.AccountId,
                 TradingConditionId = message.TradingConditionId,
                 BaseAssetId = message.BaseAssetId,
-                Balance = (double) message.Balance,
-                WithdrawTransferLimit = (double) message.WithdrawTransferLimit,
+                Balance = message.Balance,
+                WithdrawTransferLimit = message.WithdrawTransferLimit,
 
-                MarginCall = (double) message.MarginCall,
-                StopOut = (double) message.StopOut,
-                TotalCapital = (double) message.TotalCapital,
-                FreeMargin = (double) message.FreeMargin,
-                MarginAvailable = (double) message.MarginAvailable,
-                UsedMargin = (double) message.UsedMargin,
-                MarginInit = (double) message.MarginInit,
-                PnL = (double) message.PnL,
-                OpenPositionsCount = (double) message.OpenPositionsCount,
-                MarginUsageLevel = (double) message.MarginUsageLevel,
+                MarginCall = message.MarginCall,
+                StopOut = message.StopOut,
+                TotalCapital = message.TotalCapital,
+                FreeMargin = message.FreeMargin,
+                MarginAvailable = message.MarginAvailable,
+                UsedMargin = message.UsedMargin,
```

Comment:
It can be problem with "Arithmetic overflow error converting numeric to data type numeric" here.